server/api/routes/users.py

```diff
@@ -30,7 +31,7 @@ def login():
             'email': 'No account with this email exists.',
             'password': ''
         }, 404
-    if user.password != req['password']:
+    if not bcrypt.check_password_hash(user.password, req['password']):
         return {
             'email': 'An account was found but the password is incorrect.',
             'password': ''
```

Comment:
This isn't relevant to this PR, but I didn't notice earlier. This should probably be a password error saying that the password is incorrect